### ISSUE TITLE
Agree upon the usage of the latest tag

### ISSUE THREAD

**Issue by mollux** [mautic/docker-mautic#250]
There is currently some confusion about the `latest` tag on Dockerhub.

The main confusion is that https://hub.docker.com/r/mautic/mautic/tags shows the `latest` as first tag (that's a Dockerhub issue) , and that that tag points to an image with Mautic 2.16.2 (that's our issue).

This was decided when releasing the M4 Docker images, see https://github.com/mautic/docker-mautic/tree/mautic4?tab=readme-ov-file#pulling-mautic-images-from-docker-hub.

While IMO it's **not a good practice** to depend on the `latest` tag for multiple reasons (a quick search on docker latest tag should give some pointers), we should ensure that it points to something that we define as `latest`, or remove it completely.

**I would suggest (based on the common practices in other repo's), that we keep the concept of `latest` tag, and point it to the last stable patch release in the last minor and major release, in the preferred variant of the image, being `apache`.**

Practical example:
* currently, `latest` would point to `5.0.2-apache`
* let's say we release 5.1.0 tomorrow, `latest` will point to `5.1.0-apache`
* let's say we release `5.0.3` next week,  `latest` will still point to `5.1.0-apache`, as 5.0 is not the last major and minor release

any objections against this?
I'll tag some people in the comments that I know of are involved in the Docker images, but please give your 2c if you have any

**Comment by mollux** [comment]
tagging some people for awareness: @luizeof @wget @ilkkao @escopecz @RCheesley @mabumusa1

**Comment by patrykgruszka** [comment]
Yes, it's confusing with the M2 as latest. Let's address that; your suggestion is a good one.

**Comment by ilkkao** [comment]
Agreed, normally latest points to the very latest available version of the app. No reason to deviate from that standard I think.

**Comment by mollux** [comment]
thansk for the feedback, let's proceed.

I'll update the `latest` tag together with the release of Mautic 5.0.3

**Comment by mollux** [comment]
hereby the latest tags points to what we agreed upon! https://hub.docker.com/r/mautic/mautic/tags
